content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -93,6 +109,13 @@ Product-specific dynamic text values are as follows:
 - :guilabel:`tax_excl_price`: Tax Excluded Price
 - :guilabel:`tax_incl_price`: Tax Included Price
 
+After uploading a PDF, you can then :guilabel:`Configure dynamic fields`. This will allow you to map
+any field name found in your PDF to the field you want to show by writing down any existing path.
+Headers and footers starts from the current :guilabel:`sale_order` model, whereas product document
```

Comment:
I'm not quite sure what this sentence is referring to. I cannot find sale_order or sale_order_line in the Form Fields page for a header.